CHANGES.rst

```diff
@@ -55,6 +55,7 @@ astropy.samp
 
 astropy.stats
 ^^^^^^^^^^^^^
+- ``bayesian_blocks`` no longer breaks when called with the ncp_prior input. [#8317]
```

Comment:
Need to move this entry to 2.0.12 section. Also please include this PR number in change log:
```suggestion
- ``bayesian_blocks`` no longer breaks when called with the ncp_prior input. [#8317, #8339]
```